content/develop/data-types/probabilistic/configuration.md

```diff
@@ -0,0 +1,166 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Redis probabilistic data structures support multiple configuration parameters.
+linkTitle: Configuration
+title: Configuration Parameters
+weight: 100
+---
+{{< note >}}
+As of Redis Community Edition 8.0, configuration parameters for the probabilistic data structures are now set in the following ways:
+* At load time via your `redis.conf` file.
+* At run time (where applicable) using the [`CONFIG SET`]({{< relref "/commands/config-set" >}}) command.
+{{< /note >}}
+
+
+## Redis probabilistic data structure configuration parameters
+
+The following table summarizes which Bloom filter configuration parameters can be set at run-time, and compatibility with Redis Software and Redis Cloud
+
+| Old parameter name | New parameter name                          | Run-time           | Redis<br />Software | Redis<br />Cloud |
+| :-------           | :-------                                    | :-------           | :-------              | :-------         |
+| ERROR_RATE         | [bf-error-rate](#bf-error-rate)             | :white_check_mark: | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Supported">&#x2705; Flexible & Annual</span><br /><span title="Not supported"><nobr>&#x274c; Free & Fixed</nobr></span> |
```

Comment:
All probabilistic configuration parameters will be available on Redis Software and Redis Cloud once we have featureset 8 there.

Reply:
Will featureset 8 ship after CE 8? If so, then I'll leave this as is, and I'll update when it does ship.